FinniversKit/Sources/Components/Label/Label.swift

```diff
@@ -74,3 +74,28 @@ extension Label {
         UIMenuController.shared.setMenuVisible(true, animated: true)
     }
 }
+
+// MARK: - Support HTML
+
+public extension Label {
+    func setText(fromHTMLString htmlString: String, style: [String: String] = [:]) {
+        var styledText = htmlString
+        for (itemIndex, styleItem) in style {
```

Comment:
`itemIndex` is not an index, so for clarity I think this could be renamed to i.e. `stylePlaceholder`, `styleIdentifier` or something similar